### DISCUSSION TITLE
How about Navigation when using React Native Web?

### DISCUSSION THREAD

**Discussion by sarahsidek** [necolas/react-native-web#2599] [Help]
can anyone give me some tips? it is need to use the latest version such as react navigation version 6?

**Comment by Dovakeidy** [comment]
React navigation has a dedicated space on this topic:

[React Navigation on the Web](https://reactnavigation.org/docs/web-support/)

I advise you to look first at the part entitled:  [URL integration in browser](https://reactnavigation.org/docs/configuring-links)

Do you need anything else?
